packs/data/backgrounds.db/legacy-of-the-hunter.json

```diff
@@ -0,0 +1,46 @@
+{
+    "_id": "qzKT7L1qldz14q8M",
+    "img": "systems/pf2e/icons/default-icons/background.svg",
+    "name": "Legacy of the Hunter",
+    "system": {
+        "boosts": {
+            "0": {
+                "value": [
+                    "cha",
+                    "dex"
+                ]
+            },
+            "1": {
+                "value": [
+                    "cha",
+                    "con",
+                    "dex",
+                    "int",
+                    "str",
+                    "wis"
+                ]
+            }
+        },
+        "description": {
+            "value": "<p>Your introduction to the criminal life was spurred forward by a seemingly innocuous hammer that provided you with all kinds of innovative criminal ideas. It has since left your possession,<br />but every now and then you hear a faint whisper urging you toward crime.</p>\n<p>Choose two ability boosts. One must be to <strong>Dexterity</strong> or <strong>Charisma</strong>, and one is a free ability boost.</p>\n<p>You’re trained in the Thievery skill and Engineering Lore. You gain the @UUID[Compendium.pf2e.feats-srd.B9cQLRHtXoLlF0iR]{Concealing Legerdemain} skill feat.</p>"
```

Comment:
Ugh, I can't make a comment to fix this apostrophe because the `@` symbol will break due to a github bug. Also needs to have the feat added to the details.